contracts/ERC20SimpleSwap.sol

```diff
@@ -173,18 +173,18 @@ contract ERC20SimpleSwap {
     paidOut[beneficiary] = paidOut[beneficiary].add(totalPayout);
     totalPaidOut = totalPaidOut.add(totalPayout);
     /* do the actual payments */
-
-    require(token.transfer(recipient, totalPayout.sub(callerPayout)), "SimpleSwap: SimpleSwap: transfer failed");
-    /* do a transfer to the caller if specified*/
-    if (callerPayout != 0) {
-      require(token.transfer(msg.sender, callerPayout), "SimpleSwap: SimpleSwap: transfer failed");
-    }
     emit ChequeCashed(beneficiary, recipient, msg.sender, totalPayout, cumulativePayout, callerPayout);
```

Comment:
Emit in the end?

Reply:
done, but it does not actually make a difference